clipping_manager/templates/clipping_manager/books.html

```diff
@@ -0,0 +1,75 @@
+{% extends "content.html" %}
+{% load i18n urlparams %}
+{% load staticfiles sekizai_tags %}
+
+{% block content %}
+    <div class="container mb-3 ">
+        <div class="row">
+            <div class="input-group col-10 pl-0">
+                <div class="input-group-prepend">
+                    <span class="input-group-text" id="basic-addon1">
```

Comment:
Why `basic-addon1`? The ID isn't even used is it?

Reply:
Classic bootstrap copy/paste leftovers, I'll remove it